v3/postcss.config.js

```diff
@@ -1,5 +1,7 @@
+/* eslint-disable @typescript-eslint/no-require-imports */
```

Comment:
Is there a reason this can't be a single-line disable at the point of the `require`? That would be more conventional.

Reply:
Good point, I switched over.